generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -203,8 +203,8 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
         , <%= rel.relationshipFieldNamePlural %>: mapIdList(values.<%= rel.relationshipFieldNamePlural %>)
         <%_ }) _%>
       }
-      <%_ if (uniqueRelationFields.has('users')) { _%>
-      entity.user = values.user;
+      <%_ if (uniqueRelationFields.has('users') && dto !== 'mapstruct') { _%>
```

Comment:
I don't get it, why `dto !== 'mapstruct'`?

Reply:
With DTOs, it uses a `userId` field rather than `user`.  It's not needed